search-solr/src/main/java/edu/unc/lib/boxc/search/solr/services/SearchStateFactory.java

```diff
@@ -446,6 +446,11 @@ private void populateSearchStateAdvancedSearch(SearchState searchState, Map<Stri
             searchState.getSearchFields().put(SearchFieldKey.CONTRIBUTOR_INDEX.name(), parameter);
         }
 
+        parameter = getParameter(request, searchSettings.searchFieldParam(SearchFieldKey.CONTRIBUTOR_INDEX.name()));
```

Comment:
This appears to be a duplicate of the block right before it. Was this supposed to be `SearchFieldKey.CREATOR_CONTRIBUTOR`?

Reply:
Actually, I think it's not needed at all in this case.